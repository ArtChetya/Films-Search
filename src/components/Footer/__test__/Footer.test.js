import { shallow } from 'enzyme/build';
import React from 'react';
import { Footer } from '..';

describe('Footer component', () => {
    it('should be rendered correctly', () => {
        const component = shallow(<Footer />);
        expect(component).toMatchSnapshot();
    });
});
