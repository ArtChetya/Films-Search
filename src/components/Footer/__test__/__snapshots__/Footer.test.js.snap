// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should be rendered correctly 1`] = `ShallowWrapper {}`;
